PivovarAndrew/hometask-6/main.rb

```diff
@@ -0,0 +1,30 @@
+# !/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './model/scrapper'
+require 'rspec'
+
+# Main class
+class Main
+  WEB_SITE_INFO = {
+    ADDRESS: 'https://www.onliner.by/',
+    CSS_CLASSES: {
+      SECTION: '.widget-item',
+      H: '.b-tile-header',
+      IMG: 'picture img',
+      P: '.b-tile-excerpt'
+    }
+  }.freeze
```

Comment:
please separate all constants, I'll explain on a call
lika main_url, image_css, etc.

Reply:
move constant where it will be use